docs/templates/component.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: <component name>
+caption: <short purpose of component> (optional)
+figma: <link to figma> (optional)
+---
+
+<!-- longer description about capabilities and usage of the component -->
+
+```tsx onlyCode
+import {} from '@marigold/';
+```
+
+## Examples
+
+<!-- show some cases and scenarios in which the component is used -->
+
+### Use case
```

Comment:
Use case should be one or two cases or what do you think about it?

Reply:
This is only meant as a placeholder, it's "examples" -> "primary button" for example and the latter is the "use case"